packages/mandb.rb

```diff
@@ -48,14 +62,23 @@ def self.install
       'man/man8/accessdb.man8',
       'man/man8/mandb.man8',
       'tools/chconfig'
-    ].each { |file|
+    ].each do |file|
       system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/cache/man,g' #{file}"
-    }
+    end
     system "sed -i 's,/usr/share/man,#{CREW_PREFIX}/share/man,g' tools/chconfig"
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system "sed -i 's,/usr/share/man,#{CREW_PREFIX}/share/man,g' #{CREW_DEST_PREFIX}/etc/man_db.conf"
+    system "sed -i 's,/usr/man,#{CREW_PREFIX}/share/man,g' #{CREW_DEST_PREFIX}/etc/man_db.conf"
```

Comment:
Can the sed lines be moved into `def self.patch`?

Reply:
done